tests/anisotropic_viscosity.cc

```diff
@@ -1,8 +1,594 @@
-#include <aspect/material_model/simple.h>
 #include <aspect/simulator_access.h>
 
+#include <aspect/material_model/simple.h>
+#include <aspect/material_model/grain_size.h>
```

Comment:
Why do you need grain_size.h?

Reply:
Because the anisotropic shear heating plugin needs the additional `DislocationViscosityOutputs` defined in grain_size.h.